elastic/kibana
# [Reporting] Update to puppeteer 22.8.1

## Summary

Quarterly update for puppeteer, the following change set updates puppeteer to version `22.8.1`. 

The chromium version required for this version of puppeteer is `124.0.6367.201` from revision `1274542`, as such the chromium binary included for windows and darwin platforms either match or were the closest revision to the expectation. The linux headless binary was built from commit `46cf136d27d50afd9c618d164a3b95b3b62d0027` of the same revision.

 ### Checklist







### How to verify headless build on linux
- clone the following repo https://github.com/tsullivan/kibana-dev-docker
- pull this particular PR
- follow the steps outlined in the repo, replacing any occurrence of `kibana-8.13.0-SNAPSHOT-linux-aarch64.tar.gz` from the repo above's step with the output of running build on this changeset.
- before running step 4, modify the `kibana.yml` file from the `kibana-dev-docker` repo and include the following so we might be able to verify the version of chromium running; 
   ```yaml
     logging.loggers:
       - name: plugins.reporting
         level: debug
   ```
- complete the steps outlined in the README, you'll have a linux distro of kibana running on port `5601`
- Attempt creating exports of PDF and PNG reports, in dashboard, canvas, and visualizations, on report creation attempt we would see a log output that prints out the chromium version similar to this;
<img width="1581" alt="Screenshot 2024-05-14 at 10 27 52" src="https://github.com/elastic/kibana/assets/7893459/a13de1a1-c77e-44d9-bf74-7417b7d6cfd3">
